web/client/plugins/Timeline.jsx

```diff
@@ -71,7 +71,7 @@ const TimelinePlugin = compose(
             setPlaybackRange: selectPlaybackRange,
             moveRangeTo: onRangeChanged
         }),
-    branch(({ visible = true, layers = [] }) => !visible || Object.keys(layers).length === 0, renderNothing),
+    branch(({ visible = true, layers = [] }) => !visible || Object.keys(layers).length === 0 || layers[0].type === 'wmts', renderNothing),
```

Comment:
A solution like this works only for the first layer

Reply:
Noted, lemme reimplement